moped-database/migrations/1712698310409_update_project_tags/down.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM moped_tags WHERE slug IN ('map_signals', 'map_srts', 'map_transit_enhancement', 'map_transit_speed_reliability', 'map_vision_zero_major_safety', 'map_vision_zero_ped_safety', 'map_vision_zero_rapid_response_small_scale', 'map_vision_zero_speed_management', 'map_vision_zero_street_lighting', 'map_vision_zero_systemic_safety', 'map_vision_zero_systemic_safety_curves', 'map_bikeways_2024', 'map_bikeways_2025', 'map_signals_2025', 'map_srts_2025', 'map_transit_enhancement_2025', 'map_transit_speed_reliability_2025', 'map_vision_zero_major_safety_2025', 'map_vision_zero_major_safety_2025', 'map_vision_zero_rapid_response_small_scale_2025', 'map_vision_zero_speed_management_2025', 'map_vision_zero_street_lighting_2025', 'map_vision_zero_systemic_safety_2025', 'map_vision_zero_systemic_safety_curves_2025');
```

Comment:
was the snag you ran into, that when you ran the down migrations and then tried to apply the up migrations again, there was an error? because thats what happened to me. I was able to run the down migrations fine tho.

Reply:
oh yep! i couldn't go back up after the down migration. works great now!